macaroons/constraints.go

```diff
@@ -216,3 +216,25 @@ func HasCustomCaveat(mac *macaroon.Macaroon, customCaveatName string) bool {
 
 	return false
 }
+
+// GetCustomCaveatCondition returns the custom caveat condition for the given
+// custom caveat name from the given macaroon.
+func GetCustomCaveatCondition(mac *macaroon.Macaroon, customCaveatName string) (string, error) {
```

Comment:
nit: line length, here and in several other places of the PR. Make sure you use 8 spaces for the tab character in your IDE.

Reply:
Gotcha, I'm surprised this wasn't caught by `go fmt`?

Reply:
We can't enforce that with `go fmt` since in some places we do need longer lines. That's why it's currently only in the contribution checklist: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#contribution-checklist

Reply:
Thanks I'll keep that in mind.